content/en/docs/installation/kubernetes.md

```diff
@@ -131,26 +118,59 @@ Add the Jetstack Helm repository.
 $ helm repo add jetstack https://charts.jetstack.io
 ```
 
-Update your local Helm chart repository cache.
+Update your local Helm chart repository cache:
+
 ```bash
 $ helm repo update
 ```
 
-Install the cert-manager Helm chart.
+cert-manager requires a number of CRD resources to be installed into your
+cluster as part of installation.
+
+This can either be done manually, using `kubectl`, or using the `installCRDs`
+option when installing the Helm chart.
+
+**Option 1: installing CRDs with `kubectl`**
+
+Install the `CustomResourceDefinition` resources using `kubectl`:
+
+```bash
+# Kubernetes 1.15+
+$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.15.0/cert-manager.crds.yaml
+
+# Kubernetes <1.15
+$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.15.0/cert-manager-legacy.crds.yaml
+```
+
+> **Note**: If you're using a Kubernetes version below `v1.15` you will need to install the legacy version of the CRDs.
+> This version does not have API version conversion and only supports `cert-manager.io/v1alpha2` API resources. 
+
+**Option 2: install CRDs as part of the Helm release**
+
+To automatically install and manage the CRDs as part of your Helm release, you
+must add the `--set installCRDs=true` flag to your Helm installation command.
+
+Uncomment the relevant line in the next steps to enable this.
+
+---
+
+To install the cert-manager Helm chart:
+
 ```bash
 # Helm v3+
 $ helm install \
   cert-manager jetstack/cert-manager \
   --namespace cert-manager \
-  --version v0.15.0
+  --version v0.15.0 \
+  # --set installCRDs=true
 
 # Helm v2
 $ helm install \
   --name cert-manager \
   --namespace cert-manager \
   --version v0.15.0 \
-  jetstack/cert-manager
-
+  jetstack/cert-manager \
+  # --set installCRDs=true
```

Comment:
Did we test this with Helm 2? I do remember we specifically targeted Helm 3 with this.

Reply:
We're pretty much falling back on Helm's default behaviour by including the CRDs as we do - we don't use the `crd-install` annotation, and the way that Helm handles CRDs in the `templates/` directory has not changed between 2 and 3 :).

Reply:
okay!